docs/reference/gnoclient/client.md

```diff
@@ -0,0 +1,148 @@
+func (c *Client) Call(cfg BaseTxCfg, msgs ...MsgCall) (*ctypes.ResultBroadcastTxCommit, error)
+```
+
+`Call` executes a one or more [MsgCall](#type-msgcall) calls on the blockchain.
+
+### func \(\*Client\) [Send](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_txs.go#L176>)
+
+```go
+func (c *Client) Send(cfg BaseTxCfg, msgs ...MsgSend) (*ctypes.ResultBroadcastTxCommit, error)
+```
+
+`Send` executes a one or more [MsgSend](#type-msgsend) calls on the blockchain.
+
+### func \(\*Client\) [Run](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_txs.go#L112>)
+
+```go
+func (c *Client) Run(cfg BaseTxCfg, msgs ...MsgRun) (*ctypes.ResultBroadcastTxCommit, error)
+```
+
+`Run` executes a one or more MsgRun calls on the blockchain.
+
+### func \(*Client\) [QEval](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_queries.go#L108>)
+
+```go
+func (c *Client) QEval(pkgPath string, expression string) (string, *ctypes.ResultABCIQuery, error)
+```
+
+`QEval` evaluates the given expression with the realm code at `pkgPath`.
+The `pkgPath` should include the prefix like `gno.land/`. The expression is 
+usually a function call like `Render("")`.
+
+### func \(*Client\) [Query](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_queries.go#L22>)
+
+```go
+func (c *Client) Query(cfg QueryCfg) (*ctypes.ResultABCIQuery, error)
+```
+
+`Query` performs a generic query on the blockchain.
+
+### func \(*Client\) [QueryAccount](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_queries.go#L39>)
+
+```go
+func (c *Client) QueryAccount(addr crypto.Address) (*std.BaseAccount, *ctypes.ResultABCIQuery, error)
+```
+
+`QueryAccount` retrieves account information for a given address.
+
+### func \(*Client\) [QueryAppVersion](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_queries.go#L65>)
+
+```go
+func (c *Client) QueryAppVersion() (string, *ctypes.ResultABCIQuery, error)
+```
+
+`QueryAppVersion` retrieves information about the Gno.land app version.
+
+### func \(*Client\) [Render](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_queries.go#L85>)
+
+```go
+func (c *Client) Render(pkgPath string, args string) (string, *ctypes.ResultABCIQuery, error)
+```
+
+`Render` calls the Render function for pkgPath with optional args. The `pkgPath`
+should include the prefix like `gno.land/`. This is similar to using a browser
+URL `<testnet>/<pkgPath>:<args>` where `<pkgPath>` doesn't have the prefix like
+`gno.land/`.
+
+## type [BaseTxCfg](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_txs.go#L27-L33>)
+
+`BaseTxCfg` defines the base transaction configuration, shared by all message
+types.
+
+```go
+type BaseTxCfg struct {
+    GasFee         string // Gas fee
+    GasWanted      int64  // Gas wanted
+    AccountNumber  uint64 // Account number
+    SequenceNumber uint64 // Sequence number
```

Comment:
Do you think it would make sense to specify where to find the account and sequence numbers?

Reply:
I thought about this, the thing is that sequence number & account number do not show up in our docs anywhere, and I'm not really sure where to put them at this point 🥲

Reply:
I've added a short explanation on how to fetch and use account number & sequence in the how-to guide, until we have an actual section for this.